src/main/java/dk/bec/polonez/reservationsystem/dto/offerDto/DiscountDto.java

```diff
@@ -0,0 +1,32 @@
+package dk.bec.polonez.reservationsystem.dto.offerDto;
```

Comment:
**advice** Uppercase in package names is incorrect according to Java convention.

https://docs.oracle.com/javase/tutorial/java/package/namingpkgs.html

"Naming Conventions
Package names are written in all lower case to avoid conflict with the names of classes or interfaces."

Reply:
we will change it later because in order to avoid changing it by everyone